src/process.cr

```diff
@@ -129,19 +129,32 @@ class Process
     pid
   end
 
-  # The standard `IO` configuration of a process:
-  #
-  # * `nil`: use a pipe
-  # * `false`: no `IO` (`/dev/null`)
-  # * `true`: inherit from parent
-  # * `IO`: use the given `IO`
-  alias Stdio = Nil | Bool | IO
+  # How to redirect the standard input, output and error IO of a process.
+  enum Redirect
+    # Pipe the IO so the parent process can read (or write) to the process IO
+    # throught `#input`, `#output` or `#error`.
+    Pipe
+
+    # Discards the IO.
+    Close
+
+    # Use the IO of the parent process.
+    Inherit
+  end
+
+  PIPE    = Redirect::Pipe
```

Comment:
Personally, I'd consider removing these constants, `stdin: Process::Redirect::Pipe` is more explanatory to me and looks nicer than `stdin: Process::PIPE`. I don't like the inconsistency between the enum constants and the constants on `Process`.